packages/protocol/contracts/common/UsingRegistryV2WithStorageSlot.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.5.13;
+
+import "./UsingRegistryV2.sol";
+
+contract UsingRegistryV2WithStorageSlot is UsingRegistryV2 {
+  IRegistry public registry;
```

Comment:
I'm assuming this can't be renamed to avoid confusion? Maybe we could still add a comment specifying that this 'registry' variable should never be referenced and is just a place holder

Reply:
yes unfortunately renaming this breaks release tooling 😞 will add a comment!